client/connect-account-page/index.js

```diff
@@ -2,19 +2,81 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
 import { Card } from '@woocommerce/components';
 import { Button, Notice } from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { useState, useEffect } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import Page from 'components/page';
-import HeroImage from './hero-image';
 import strings from './strings';
 import wcpayTracks from 'tracks';
+import HeroImage from './hero-image.js';
+import WCPayLogo from './wcpay-logo.js';
+import Visa from './cards/visa.js';
+import MasterCard from './cards/mastercard.js';
+import Maestro from './cards/maestro.js';
+import Amex from './cards/amex.js';
+import ApplePay from './cards/applepay.js';
+import CB from './cards/cb.js';
+import DinersClub from './cards/diners.js';
+import Discover from './cards/discover.js';
+import JCB from './cards/jcb.js';
+import UnionPay from './cards/unionpay.js';
+
+const Masthead = () => (
+	<header className="wcpay-connect-account-page-masthead">
+		<h1>
+			<WCPayLogo />
+		</h1>
+		<div className="wcpay-connect-account-page-recommended">
+			<Pill>{ strings.recommended }</Pill>
+		</div>
+		<div className="wcpay-connect-account-page-hero">
+			<HeroImage />
+		</div>
+	</header>
+);
+
+const Pill = ( props ) => (
+	<div className="wcpay-connect-account-page-pill">{ props.children }</div>
+);
+
+const LearnMore = () => {
+	const handleClick = () => {
+		wcpayTracks.recordEvent(
+			wcpayTracks.events.CONNECT_ACCOUNT_LEARN_MORE
+		);
+	};
+	return (
+		<a onClick={ handleClick } href="https://woocommerce.com/payments/">
```

Comment:
According to the parent issue, this needs to be opened in a new tab

Reply:
Thanks! Fixed in d11d457